### PR TITLE
- Added dbot score to ip+domain commands

### PR DESCRIPTION
**Pull request by reutshal**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18430

## Description
- Added DbotScore calculation
- Updated outputs accordingly 
- Added test playbook

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [X] Code Review

### CODE REVIEW

File: Integrations/integration-Threat_Crowd_CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## [Unreleased]
+
+ - Added DbotScore calculation to the following commands:
+   - threat-crowd-ip
+   - thrat-crowd-domain
+ - Updated outputs accordingly
```
Comment (ronykoz) [inline Integrations/integration-Threat_Crowd_CHANGELOG.md]: remove this line

File: Integrations/integration-Threat_Crowd_CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+ - Added test playbook
```
Comment (ronykoz) [inline Integrations/integration-Threat_Crowd_CHANGELOG.md]: not relevant to the RN

File: Integrations/integration-Threat_Crowd.yml [hunk 3]
```diff
@@ -166,6 +182,48 @@ script:
         delete(args.file);
     }
 
+    function calculateDBotScore(res, args, commandData) {
+        var dbotScore = 0;
+        if (res.response_code == 1){
```
Comment (ronykoz) [inline Integrations/integration-Threat_Crowd.yml]: should it be 1? seems strange, maybe 200?

File: Integrations/integration-Threat_Crowd.yml [hunk 4]
```diff
@@ -116,13 +118,27 @@ script:
                 }
                 current[values[j]] = obj[k];
             });
+
+            if (dbotScore == 3) {
+                res.Malicious = {
+                   "Vendor": 'Threat Crowd',
+                   "Description": 'Votes field is -1'
```
Comment (ronykoz) [inline Integrations/integration-Threat_Crowd.yml]: I think we should have a better explanation here

**Comment by ronykoz** [comment]
@kirbles19 could you go over the following files:
https://github.com/demisto/content/blob/threatCrowd-DbotScore/Integrations/integration-Threat_Crowd.yml

https://github.com/demisto/content/blob/threatCrowd-DbotScore/Integrations/integration-Threat_Crowd_CHANGELOG.md

and also checkout the description for the malicious section, i think it should be clarified `"Description": 'Votes field is -1'`

**Comment by ygerowitz** [comment]
@reutshal The second comment in the list says **thrat-crowd-domain.** "thrat" is misspelled (it should be "threat".
The second is regarding *- Updated outputs accordingly*. Is the word "accordingly" required here? Unless it refers to some specific updates that are not mentioned in this particular text, I think it can be left out.

**Comment by ygerowitz** [comment]
@reutshal The Threat Crowd API documentation describes **-1** as **Most users have voted this malicious**.

**Comment by ygerowitz** [comment]
Changed **Votes field is -1** to **Most users have voted this entity malicious** in https://github.com/demisto/content/blob/threatCrowd-DbotScore/Integrations/integration-Threat_Crowd.yml.

**Comment by ronykoz** [comment]
@reutshal lets finish going over my comments and wait for @ygerowitz approval and get this PR merged

File: Integrations/integration-Threat_Crowd.yml (hunk 4 above)
Comment (reutshal) [reply Integrations/integration-Threat_Crowd.yml]: Changed

File: Integrations/integration-Threat_Crowd.yml (hunk 3 above)
Comment (reutshal) [reply Integrations/integration-Threat_Crowd.yml]: res here is the body of the result that came back from http method.
this api returns 0 in failure and 1 in success   
 
- success:
image

- failed request or no results :
image

**Comment by ygerowitz** [comment]
Replaced both instances of **-1** with **Most users have voted this entity malicious.**

**Comment by ygerowitz** [comment]
Regarding the md file, I have not changed the typo in the command name.

## [Unreleased]

- Added DbotScore calculation to the following commands.
  - threat-crowd-ip
  - *thrat*-crowd-domain
I have removed the lines following the bullets.

**Pull request merged**